catalog/ui/catalog-ui-search/src/main/webapp/js/view/ingest/IngestModal.view.js

```diff
@@ -117,14 +117,17 @@ define([
                         type = 'JAR';
                     }
                 }
+                var parsedProgress = parseInt(data.loaded / data.total * 100, 10);
```

Comment:
I _think_ you can do `var parsedProgress = parseInt(...) || 0`; and skip the second line.

Also, is the `NaN` caused by `data.total` being `0`, or because of something else?